packages/application-generic/src/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -208,10 +209,16 @@ Handlebars.registerHelper(
 export class CompileTemplate {
   async execute(command: CompileTemplateCommand): Promise<string> {
     const templateContent = command.template;
-
-    const template = Handlebars.compile(templateContent);
-
-    const result = template(command.data, {});
+    let result = '';
+    try {
+      const template = Handlebars.compile(templateContent);
+
+      result = template(command.data, {});
+    } catch (e: any) {
```

Comment:
```suggestion
    } catch (e: unknown) {
```
Better TS ;)

Reply:
Unfortunately, it creates a ts error for e.message 😬

Reply:
At the moment, thats how all catch errors are typed in the project

Reply:
@ainouzgali You can use `checkIsResponseError` as a type-guard here! TypeScript began recommending `unknown` as the type for `catch` in TS 4.4

Reply:
@BiswaViraj Side note: I would suggest that we reserve `Request changes` for blocking problems (like `issue` or important `chore`s) 🙂 -- this is probably more of a nitpick until we put it into a coding style guide

Reply:
Cool. Didn't know about it. Thanks!